### DISCUSSION TITLE
[Question] How to sync 2 plex server ?

### DISCUSSION THREAD

**Discussion by RobertusIT** [arabcoders/watchstate#273] [General] [answered]
Hi,

i have 2 plex server in different location. ( 2 different city )

I don't understand how to do that.

1 - I need to create a docker compose, in local, for every plex server? Inside plex ? ( I use proxmox and a lxc for plex )

2 -  How can i import from a plex server to another, if aren't in the same location ?

**Comment by arabcoders** [comment, marked as answer]
Hi,

You only need to have the docker running once you dont need to run it on same servers or anything, after creating your docker container, simply add the two servers and let it do the syncing, please see the [README.md](/ArabCoders/watchstate/blob/master/README.md) and [FAQ.md](/ArabCoders/watchstate/blob/master/FAQ.md), If you need further help please reply to this thread.

**Comment by RobertusIT** [comment]
I have a quwstion

because i have 2 servers that have the same library and same path.

1 - Watchstate, sync anyway between servers, also partial watched?
2 - If i play a movie from Server 1 and I don't see until the end, and tomorrow Server 1 is down. Can I have in the Plex dashboard in Continue Watching, this movie? 
3 - And play this movie, without error? So i don't need to search again and select Server 2.

I ask it, to find a solution, when a server is down, users can't lost partial watched, have the same continue watching, so have the same experience, can't see that a server is down.

**Reply by arabcoders** [reply]
Partial watch is something we would like to support in the future. Currently its not supported due to limitations on the method we choose to implement the sync process. 

Sadly i was going to suggest plex own watch state feature but it also doesn't support it. its really hard to do via the API. 

I have some ideas i would like to explore in the future to hopefully support this But nothing i can share at the moment.
